demo/meeting/src/containers/LocalMediaStreamMetrics/index.tsx

```diff
@@ -0,0 +1,176 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import React from 'react';
+
+import {
+  useMediaStreamMetrics,
+  useAudioVideo,
+  PopOverHeader
+} from 'amazon-chime-sdk-component-library-react';
+
+import MediaStatsList from '../../components/MediaStatsList/index';
+import MetricItem from '../../components/MediaStatsList/MetricItem';
+import { StyledMediaMetricsWrapper } from '../../components/MediaStatsList/Styled';
+
+export const LocalMediaStreamMetrics: React.FC = () => {
+  const audioVideo = useAudioVideo();
+  const localAttendeeId = (audioVideo as any).audioVideoController
+    ?.realtimeController?.state?.localAttendeeId;
+  const {
+    audioPacketsSentFractionLossPercent,
+    audioPacketsReceivedFractionLossPercent,
+    availableIncomingBandwidth,
+    availableOutgoingBandwidth,
+    videoStreamMetrics
+  } = useMediaStreamMetrics();
+  const localVideoStreamMetrics = videoStreamMetrics[localAttendeeId];
+  const ssrcArray = localVideoStreamMetrics
```

Comment:
The `localVideoStreamMetrics` is `{}` minimally without it being filled right or does it return a falsy value here like `null`, or `undefined`?

Because, if it does not then `{}` is still considered truthy and `Object.keys({})` will be `[]`. So, no need for tertiary here I think.

Reply:
Gotcha. Will remove the tertiary here. Thanks for pointing it out.